apps/fz_http/lib/fz_http/telemetry.ex

```diff
@@ -79,7 +79,16 @@ defmodule FzHttp.Telemetry do
         [
           user_count: Users.count(),
           device_count: Devices.count(),
-          users_with_mfa: MFA.count_distinct_by_user_id()
+          max_devices_for_users: Devices.max_count_by_user_id(),
+          users_with_mfa: MFA.count_distinct_by_user_id(),
+          users_with_mfa_totp: MFA.count_distinct_totp_by_user_id(),
```

Comment:
I think these two are the same right? I think we only support TOTP right now. Not suggesting to change, it's better to start collecting telemetry earlier than later, just wondering if you were expecting these to be different.

Reply:
I was expecting these to be different, because I saw different types for the MFA methods.